Build for ppc64le

weiji14 opened the pull request:
Xref https://conda-forge.org/docs/maintainer/conda_forge_yml/#build-platform. ~Had to switch away from `pixi` to workaround https://github.com/conda-forge/conda-smithy/issues/2350. Also switched to native build for aarch64~ Edit: can keep previous cross-compile mode setup with pixi.

Should fix the build error at https://github.com/conda-forge/prek-feedstock/pull/1#issuecomment-3289957894

<!--
Thank you for your pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->

weiji14 committed:
Build for ppc64le

Xref https://conda-forge.org/docs/maintainer/knowledge_base/#emulated-builds. Also switch to native build for aarch64

weiji14 committed:
MNT: Re-rendered with conda-smithy 3.52.2 and conda-forge-pinning 2025.09.14.19.31.04

Other tools:
- conda-build 25.7.0
- rattler-build 0.47.0
- rattler-build-conda-compat 1.4.6

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/recipe.yaml```) and found it was in an excellent condition.

weiji14 committed:
Don't use pixi as not available for ppc64le

Got error `error: 'https://github.com/prefix-dev/pixi/releases/latest/download/pixi-ppc64le-unknown-linux-musl.tar.gz' is not available`, xref https://github.com/conda-forge/conda-smithy/issues/2350

weiji14 committed:
MNT: Re-rendered with conda-smithy 3.52.2 and conda-forge-pinning 2025.09.14.19.31.04

Other tools:
- conda-build 25.7.0
- rattler-build 0.47.0
- rattler-build-conda-compat 1.4.6

xhochy commented:
What's the reason for the emulated build here? I don't like moving to this as drastically slows down CI.

weiji14 commented:
Can ppc64le be built in cross-compile mode? From reading https://conda-forge.org/docs/maintainer/knowledge_base/#emulation-examples, I thought it had to be emulation only. Feel free to change it to cross-compile if that's possible.

xhochy commented:
Sure, it can be built in cross-compilation. That's the most dominant mode actually.

weiji14 committed:
Switch from emulated to cross-compile mode

weiji14 committed:
MNT: Re-rendered with conda-smithy 3.52.2 and conda-forge-pinning 2025.09.15.11.24.57

Other tools:
- conda-build 25.7.0
- rattler-build 0.47.0
- rattler-build-conda-compat 1.4.6

weiji14 commented:
Oh yes, much faster now (17min -> 2min). Ready for review again!
